README.md

```diff
@@ -97,24 +89,174 @@ const YourComponent = () => {
     query: `{ todos { id } }`,
   });
 
-  const { fetching, data } = result;
-  return fetching ? <Loading /> : <List data={data.todos} />;
+  if (result.error) return 'Oh no!';
+  if (result.fetching) return 'Loading...';
```

Comment:
I think I tend to prefer when docs show some actual components rendered in situations like this than just strings, i.e.:

```typescript
if (result.error) {
   return <Error message={result.error.message} />;
}

if (result.fetching || !result.data) {
    return <Loading />;
}
```

The example above also handles the `fetching: false` but `data` false-y case – is that still an issue we're hitting?

Reply:
Good point! False-y data shouldn’t be an issue as long as the API can be trusted to not have nullable data, so that’s purely schema-dependent.

In all usual cases it’s either: fetching, error, or data, where data and error may co-exist depending on whether it’s partial GraphQL errors that don’t affect the entire query